audb/core/dependencies.py

```diff
@@ -308,29 +309,30 @@ def load(self, path: str):
         """
         self._data = {}
         path = audeer.safe_path(path)
-        if path.endswith('pkl') and os.path.exists(path):
-            df = pd.read_pickle(path, compression='xz')
-        elif path.endswith('csv') and os.path.exists(path):
-            # Data type of dependency columns
-            dtype_mapping = {
-                name: dtype for name, dtype in zip(
-                    define.DEPEND_FIELD_NAMES.values(),
-                    define.DEPEND_FIELD_DTYPES.values(),
+        if os.path.exists(path):
```

Comment:
I wonder if it would be safer to actually raise an error here instead of silently ignore that the file does not exist.